packages/account/src/Components/inline-note-with-icon/inline-note-with-icon.tsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Icon, Text } from '@deriv/components';
 
-const InlineNoteWithIcon = ({ icon, message, title }) => {
+export type TInlineNoteWithIconExtend = {
```

Comment:
Not sure why we need to export this type but if you need to use it somewhere else ( where `<InlineNoteWithIcon/>` is also being used ) you can use `React.ComponentProps<typeof InlineNoteWithIcon> ` . it will give you access to `TInlineNoteWithIconExtend` type and if you need some specific type in it you can grab them using `Pick<type>` utility.
this way you don't need to import the type along with the component